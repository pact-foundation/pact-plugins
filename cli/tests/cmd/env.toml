bin.name = "pact-plugin"
args = "env --help"
