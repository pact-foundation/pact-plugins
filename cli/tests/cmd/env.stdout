Print out the Pact plugin environment config

Usage: pact-plugin env

Options:
  -h, --help  Print help
